### PR TITLE
CREAT broadlink A2 

### PR DESCRIPTION
**Pull request by trz0332**
CREAT broadlink A2

### CODE REVIEW

**Comment by trz0332** [comment]
__init__

**Comment by felipediel** [comment]
Good job @trz0332, thank you! I'm preparing this PR for the next release. Could you give me a sample of check_sensors_raw()? And maybe a link to the device? I didn't find it anywhere.

**Comment by felipediel** [comment]
http://cn.ibroadlink.com/pagea2.html ?

File: broadlink/sensor.py [hunk 1]
```diff
@@ -45,3 +45,26 @@ def check_sensors_raw(self) -> dict:
...
+        payload = self.decrypt(response[0x38:])
+        data = payload[0x6:]
+
+        return {
+            "temperature": data[0xd]*255+ data[0xe],
+            "humidity": data[0xf]*255+ data[0x10],
+            "PM10": data[0x7]*255+ data[0x8],
+            "PM2.5": data[0x9]*255+ data[0xa],
+            "PM1.0": data[0xb]*255+ data[0xc],
+        }
```
Comment (felipediel) [inline broadlink/sensor.py:70]: Maybe * 256 instead?
```python3
        return {
            "temperature": data[0xd] * 256 + data[0xe],
            "humidity": data[0xf] * 256 + data[0x10],
            "PM10": data[0x7] * 256 + data[0x8],
            "PM2.5": data[0x9] * 256 + data[0xa],
            "PM1.0": data[0xb] * 256 + data[0xc],
        }
```

**Comment by felipediel** [comment]
Closed with https://github.com/mjg59/python-broadlink/pull/791 and https://github.com/mjg59/python-broadlink/pull/792.

Good job, @trz0332, thank you!

**Pull request closed without merging**